joomla/joomla-cms
# [com_redirect] Improve disabled redirect plugin message

#### Summary of Changes

Simple PR to improve the disable redirect system plugin message in com_redirect.

Before (message is an error and the link is to the plugin list with harcoded text search)
![image](https://cloud.githubusercontent.com/assets/9630530/15659884/832fdff8-26ca-11e6-9086-d60dbd694872.png)

After (message is a warning and the link is now to edit the redirect system plugin)
![image](https://cloud.githubusercontent.com/assets/9630530/15659940/a40adb82-26cb-11e6-8d19-f516cbaf93aa.png)
#### Testing Instructions

Code review, or:
1. Use latest staging
2. Disable redirect systme plugin
3. Go to Components -> Redirect and check the error message. Click the link, you'll be redirected to the plugin manager list view with a search.
4. Apply patch
5. Go to Components -> Redirect and check the warning message. Click the link, you'll be redirected to directly edit the redirect plugin.
